### PR TITLE
Add support for Net Standard 1.3

### PR DESCRIPTION
**Pull request by ancaantochi**

### CODE REVIEW

File: src/ProtocolGateway.Core/Instrumentation/SafePerformanceCounter.cs [hunk 1]
```diff
@@ -4,12 +4,14 @@
 namespace Microsoft.Azure.Devices.ProtocolGateway.Instrumentation
 {
     using System;
+#if !NETSTANDARD1_3
```
Comment (nayato) [inline src/ProtocolGateway.Core/Instrumentation/SafePerformanceCounter.cs]: no need for the #if now (whole file is removed from build for netstandard)

File: host/ProtocolGateway.Host.Console.NetStandard/ConsoleEventListner.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace ProtocolGateway.Host.Console.NetStandard
+{
+    using System.Diagnostics.Tracing;
+    using System.Text;
+
+    class ConsoleEventListner : EventListener
```
Comment (nayato) [inline host/ProtocolGateway.Host.Console.NetStandard/ConsoleEventListner.cs]: typo

File: host/ProtocolGateway.Host.Console.NetStandard/ProtocolGateway.Host.Console.NetStandard.csproj [hunk 3]
```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>exe</OutputType>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
+    <RuntimeFrameworkVersion>2.0.0-beta-001753-00</RuntimeFrameworkVersion>
```
Comment (nayato) [inline host/ProtocolGateway.Host.Console.NetStandard/ProtocolGateway.Host.Console.NetStandard.csproj]: >2.0.0-beta-001753-00 [](start = 29, length = 20)

this is somewhat aggressive. could you pls downgrade to LTS release?

File: src/ProtocolGateway.Core/Instrumentation/SafePerformanceCounter.cs (hunk 1 above)
Comment (ancaantochi) [inline src/ProtocolGateway.Core/Instrumentation/SafePerformanceCounter.cs]: done

File: host/ProtocolGateway.Host.Console.NetStandard/ConsoleEventListner.cs (hunk 2 above)
Comment (ancaantochi) [inline host/ProtocolGateway.Host.Console.NetStandard/ConsoleEventListner.cs]: done

File: host/ProtocolGateway.Host.Console.NetStandard/ProtocolGateway.Host.Console.NetStandard.csproj (hunk 3 above)
Comment (ancaantochi) [inline host/ProtocolGateway.Host.Console.NetStandard/ProtocolGateway.Host.Console.NetStandard.csproj]: done

**Comment by nayato** [comment]
looks good now, thanks!

**Pull request merged**